zerver/lib/actions.py

```diff
@@ -2044,6 +2045,21 @@ def do_change_default_language(user_profile, setting_value, log=True):
         log_event(event)
     send_event(event, [user_profile.id])
 
+def do_update_user_star_status(user_profile, email, status):
```

Comment:
`status` is a `bool`; am I right?
If yes, then you should annotate this function as `# type: (UserProfile, text_type, bool) -> Dict[str, bool]`.